### PR TITLE
Add tf2onnx

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/tf2onnx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true     # [win or py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+    - tensorflow >=1.5.0
```
Comment (chrisburr) [inline recipes/tf2onnx/meta.yaml]: You shouldn't need any host dependencies other than python, pip and pytest-runner

Comment (marcelotrevisani) [reply recipes/tf2onnx/meta.yaml]: yeap, I know, that was just for testing purposes.

But thanks for the follow-up! :)

**Comment by marcelotrevisani** [comment]
For some reason, there is a dependency conflict with Python 3.7

**Comment by scopatz** [comment]
Is this ready?

**Comment by marcelotrevisani** [comment]
> Is this ready?

I think so. But it is just for Python ``3.6`` for now.
I might try to create a package for Python 3.7 as well, but I try to do it on the feedstock

**Pull request merged**